src/attributes.jl

```diff
@@ -920,15 +920,23 @@ most recent call to [`compute_conflict!`](@ref).
 Possible values are:
 * `COMPUTE_CONFLICT_NOT_CALLED`: the function [`compute_conflict!`](@ref) has
   not yet been called
+* `COMPUTE_CONFLICT_FAILED`: the function [`compute_conflict!`](@ref) has been
+  called and failed to completely execute. In particular, it could not decide
+  whether a conflict exists or not
 * `NO_CONFLICT_EXISTS`: there is no conflict because the problem is feasible
-* `NO_CONFLICT_FOUND`: the solver could not find a conflict
+* `NO_CONFLICT_FOUND`: the solver could not find a conflict, but did not ensure
+  the problem is feasible
 * `CONFLICT_FOUND`: at least one conflict could be found
+* `MINIMUM_CONFLICT_FOUND`: at least one conflict could be found and the solver
+  was able to prove this conflict is minimal
 """
 @enum ConflictStatusCode begin
     COMPUTE_CONFLICT_NOT_CALLED
+    COMPUTE_CONFLICT_FAILED
     NO_CONFLICT_EXISTS
     NO_CONFLICT_FOUND
     CONFLICT_FOUND
+    MINIMUM_CONFLICT_FOUND
```

Comment:
Hmm. Couldn't we return `NO_CONFLICT_FOUND` for the timeout/limit statuses? And can't we just return `CONFLICT_FOUND` regardless of whether it is minimal? I don't think we need lots of statuses for this. It it likely going to be used infrequently for debugging purposes.

Reply:
In some cases, the solver hits the limit, but is still able to return a conflict. In that case, we would return CONFLICT_FOUND in many different situations. 

To simplify a bit, we could merge NO_CONFLICT_EXISTS and NO_CONFLICT_FOUND.

Reply:
Another solution would have "hierarchical" codes. For instance, `termination_status` (or its IIS equivalent) would only return fine-grained codes; users may rely on other functions like `has_found_optimum` or `has_ended_prematurely_but_has_a_solution` to get higher-level information. I'm not sure the advantages would overweight the costs, though…

Reply:
What are the advantages of having fine-grain statuses for the IIS? Either the solver found one, or it didn't. I don't really care if it hit a time-out and returned the one it had, even if it wasn't minimal. If people want more control, they could use the solver-dependent methods.

Reply:
In the new commit, there are only four remaining statuses: not called, error, nothing to find, something has been found.

Reply:
I guess my point still stands: when would I want to distinguish between `COMPUTE_CONFLICT_FAILED` and `NO_CONFLICT_FOUND`?

Because I should only be calling `compute_conflict` if the termination status is infeasible. So I don't think we need to account for the case where the model is feasible. `NO_CONFLICT_FOUND` should just be that. We tried finding a conflict and couldn't. That could be because of a time limit, or because the model is feasible, or something else.

Reply:
Is the new formulation loose enough?

Reply:
Yeah, I think this makes more sense to me. Although we should probably wait for someone else to chime in.

Reply:
I'm just calling those that gave an opinion on the previous PR: @joaquimg @blegat

Reply:
I can imagine that for some solver, it might be difficult to distinguish between whether it was not found because it does not exists or whether it was not found because of some other issue.
If we make it clear that `NO_CONFLICT_FOUND` does not exclude the fact that it might be because the problem is feasible then IMO we could also have `NO_CONFLICT_EXIST`. It provide some solver-independent useful information to know that it was not found not because it has a numerical issue but because there was none and it is sure of it.

Reply:
Okay, maybe I'm actually coming back round to what we have is okay. 

We can return `NO_CONFLICT_EXISTS` if the solver proved there was no conflict, and `NO_CONFLICT_FOUND` if it hit a time limit etc without finding one.

Reply:
Better this way?